gamemodes/terrortown/entities/entities/ttt_logic_role.lua

```diff
@@ -29,7 +29,7 @@ function ENT:KeyValue(key, value)
 		self.checkingRole = tonumber(value)
 
 		if not self.checkingRole then
-			ErrorNoHalt("ttt_logic_role: bad value for Role key, not a number\n")
+			ErrorNoHaltWithStack("ttt_logic_role: bad value for Role key, not a number\n")
```

Comment:
and here as well, this is also a map entity that is never triggered inside lua. This is an error the mapmaker has to take care of, hence I think the stack is useless